build.sbt

```diff
@@ -13,7 +13,7 @@ ThisBuild / licenses     := Seq(License.MIT)
 ThisBuild / developers := List(
   Developer(id = "rolang", name = "Roman Langolf", email = "rolang@pm.me", url = url("https://rolang.dev"))
 )
-ThisBuild / versionScheme := Some("semver-spec")
+ThisBuild / versionScheme := Some("early-semver")
```

Comment:
just curios, will it work with `Some(VersionScheme.Always)`?

Reply:
`early-semver` seem to be enforced by the `sbt-typelevel` plugin, getting this:
```
java.lang.IllegalArgumentException: requirement failed: Only early-semver versioning scheme supported.
        at scala.Predef$.require(Predef.scala:281)
        at org.typelevel.sbt.TypelevelMimaPlugin$.$anonfun$buildSettings$2(TypelevelMimaPlugin.scala:48)
```
Must be a standard for typelevel stack I guess to avoid binary compatibility issues?
I tend to just bump the base version from 0.1 to 0.2 instead of trying to work around it or trying to keep it binary compatible🤔

Reply:
> Must be a standard for typelevel stack I guess to avoid binary compatibility issues?

 I believe so. Recently, I delved into the [sbt sources](https://github.com/sbt/librarymanagement/blob/develop/core/src/main/scala/sbt/librarymanagement/VersionNumber.scala#L206-L248) and discovered that binary compatibility is primarily maintained through the use of suffixes.

Reply:
yes, in addition to that [sbt-typelvel](https://github.com/typelevel/sbt-typelevel?tab=readme-ov-file#features) adds [MiMa](https://github.com/lightbend/mima) checks to generated github action to confirm that you don't lie about your version being compatible :smile: